packages/sdk/src/handlers/collections.ts

```diff
@@ -33,7 +33,7 @@ export class CollectionsHandler<T = CollectionItem> {
 	}
 
 	async createMany(collections: PartialItem<T>[]): Promise<ManyItems<T>> {
-		const { data, meta } = await this.transport.get(`/collections`, {
+		const { data, meta } = await this.transport.post(`/collections`, {
```

Comment:
Also needs to post the actual `collections` data as data, not query params 🙂 
(I'll resolve that rn)

Reply:
Ah okay, thanks! 😃